libs/model/src/services/commonProtocol/launchpadHelpers.ts

```diff
@@ -0,0 +1,94 @@
+import { commonProtocol } from '@hicommonwealth/shared';
+import Web3 from 'web3';
+import { models } from '../../database';
+import { mustExist } from '../../middleware/guards';
+import { tokenCommunityManagerAbi } from './abi/TokenCommunityManager';
+import { erc20Abi } from './abi/erc20';
+
+export async function createTokenHandler(
+  chainNodeId: number,
+  tokenAddress: string,
+  description?: string,
+  iconUrl?: string,
+) {
+  const chainNode = await models.ChainNode.findOne({
```

Comment:
- we should discuss these protocol helpers in detail (retro chat probably). 
- the idea is that we don't leak models in helpers, we pass loaded entities if necessary
- models (db) should be only used inside commands or queries